src/components/Top-Bar.vue

```diff
@@ -13,12 +13,7 @@
       </div>
       <search-bar v-if="!isSearchDisabled" />
     </oc-grid>
-    <oc-grid
-      v-if="!isPublicPage"
-      flex
-      gutter="small"
-      class="uk-width-expand uk-flex-right oc-mt-rm"
-    >
+    <oc-grid v-if="!isPublicPage" flex gutter="small" class="uk-width-expand uk-flex-right oc-m-rm">
```

Comment:
Hint for reviewers: the only change here is that we don't use the utility class for removing the top margin (`oc-mt-rm`) anymore, but instead the utility class for removing all margins (`oc-m-rm`). The `oc-grid` is introducing a negative left margin (`-15px`) which is pulling parts of the topbar on top of the button.